tlatools/org.lamport.tlatools/test/tlc2/tool/SimulatorTest.java

```diff
@@ -0,0 +1,31 @@
+package tlc2.tool;
+
+import org.junit.Test;
+
+import tlc2.debug.StandaloneConstExpressionDebugger;
+import tlc2.output.EC;
+import tlc2.output.MP;
+import tlc2.tool.impl.DebugTool;
+import tlc2.util.RandomGenerator;
+import util.Assert;
+import util.SimpleFilenameToStream;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+import static org.junit.Assert.assertTrue;
+
+public class SimulatorTest extends CommonTestCase {
+
+    @Test
+    public void testPrintBehaviorShouldPrintErrorState() throws IOException {
+        MP.setRecorder(recorder);
+        Simulator simulator = new Simulator(new DebugTool(BASE_PATH + "Github726", BASE_PATH + "Github726", new SimpleFilenameToStream(),
```

Comment:
it would be nice to include some mocking library...

Reply:
We just [removed EasyMock](https://github.com/tlaplus/tlaplus/commit/2a942645b7408930841b2271bdfdabf5dd91082d). :-)